### PR TITLE
feat: per-guide attention strength control in self-attention

### PR DESCRIPTION
**Pull request by tavihalperin**
This is too allow spatial control of ic-lora strength

Implements per-guide attention attenuation via log-space additive bias in self-attention. Each guide reference tracks its own strength and optional spatial mask in conditioning metadata (guide_attention_entries).

- model.py: build composite attention mask from per-guide entries
- av_model.py: thread self_attention_mask through AV transformer blocks
- model_base.py: pass guide_attention_entries via extra_conds
- nodes_lt.py: guide entry tracking in LTXVAddGuide, cleanup in CropGuides

### CODE REVIEW

File: comfy/ldm/lightricks/model.py [hunk 1]
```diff
@@ -890,13 +909,242 @@ def _process_input(self, x, keyframe_idxs, denoise_mask, **kwargs):
...
+            device: Target device.
+            dtype: Target dtype.
+
+        Returns:
+            (1, 1, total_tokens, total_tokens) additive bias mask.
+            0.0 = full attention, negative = attenuated, finfo.min = effectively fully masked.
+        """
+        finfo = torch.finfo(dtype)
+        mask = torch.zeros((1, 1, total_tokens, total_tokens), device=device, dtype=dtype)
+        tracked_end = guide_start + tracked_count
```
[the remark this answers is not in the document]
Comment (tavihalperin) [reply comfy/ldm/lightricks/model.py:1132]: Acknowledged. Current IC-LoRA use cases are short clips (~10K tokens, ~50 MB). Will revisit with sparse representation if longer sequences are needed.
